python/neuroglancer/pipeline/task_queue.py

```diff
@@ -247,15 +276,16 @@ def purge(self):
 
             for tid in taskids:
                 self.delete(tid)
+            
```

Comment:
That would be really good. Unfortunately, `TaskQueue.list()` only returns the first 100 elements... Maybe it would be possible to lease by tag and then delete the resulting tasks as a sweet hack.